torch/_export/converter.py

```diff
@@ -89,6 +91,16 @@ def __init__(
 
         self.subgraphs: Dict[str, torch.fx.GraphModule] = {}
 
+        # Populate methods for the standard operators.
```

Comment:
Made some changes to my registry impl to incorporate your standard operator @BoyuanFeng. Let me know if you have any feedbacks. 

cc @angelayi @ydwu4